### PR TITLE
Add OIDC docs

### PR DESCRIPTION
**Pull request by car12o**
##### Summary
Cloud OIDC authentication method docs.

### CODE REVIEW

File: integrations/cloud-authentication/metadata.yaml [hunk 1]
```diff
@@ -47,3 +47,58 @@
...
+    description: |
+      ### Prerequisites
+      - Authorization Server with OIDC protocol supported
+      - A Netdata Cloud account
+      - Access to the Space as an administrator
+      - Space needs to be on the Business plan or higher
+
+      ### Setting up Authorization Server
+      Your server should follow the [full specification for OIDC](https://openid.net/specs/openid-connect-core-1_0.html).
+      In order to integrate your Authorization Server with Netdata a creation of a client is required, clients are applications and services that can request authentication of a user.
```
Comment (hugovalente-pm) [inline integrations/cloud-authentication/metadata.yaml:74]:
```suggestion
      In order to integrate your Authorization Server with Netdata the creation of a client is required. Clients are applications and services that can request authentication of a user.
```

File: integrations/cloud-authentication/metadata.yaml [hunk 2]
```diff
@@ -47,3 +47,58 @@
...
+      The access settings for your client are the following:
```
[the remark this answers is not in the document]
Comment (car12o) [reply integrations/cloud-authentication/metadata.yaml:75]: We removed it as a suggestion from the Okta reviewer.

image

**Review by hugovalente-pm — DISMISSED** [review]

File: integrations/cloud-authentication/metadata.yaml [hunk 3]
```diff
@@ -47,3 +47,58 @@
...
 
       If you start your authentication flow from Netdata sign-in page please check [these steps](https://github.com/netdata/netdata/blob/master/docs/netdata-cloud/enterprise-sso-authentication.md#from-netdata-sign-up-page).
+
+- id: 'oidc-authentication'
+  meta:
+    name: 'OIDC'
+    link: 'https://netdata.cloud'
+    categories:
+      - auth
+    icon_filename: 'oidc.png'
```
Comment (hugovalente-pm) [inline integrations/cloud-authentication/metadata.yaml:57]:
```suggestion
    icon_filename: 'openid.svg'
```

**Comment by hugovalente-pm** [comment]
for the icon on website https://github.com/netdata/website/pull/592

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**